src/StreamJsonRpc/ProxyGeneration.cs

```diff
@@ -294,8 +294,8 @@ internal static TypeInfo Get(Type contractInterface, ReadOnlySpan<Type> addition
                     string rpcMethodName = methodNameMap.GetRpcMethodName(method);
                     if (rpcInterfaceCode.HasValue)
                     {
-                        methodName = rpcInterfaceCode + "." + method.Name;
-                        rpcMethodName = rpcInterfaceCode + "." + rpcMethodName;
+                        methodName = $"{rpcInterfaceCode.GetValueOrDefault()}.{method.Name}";
+                        rpcMethodName = $"{rpcInterfaceCode.GetValueOrDefault()}.{rpcMethodName}";
                     }
```

Comment:
```csharp
                    if (rpcInterfaceCode is { } localRpcInterfaceCode)
                    {
                        methodName = $"{localRpcInterfaceCode}.{method.Name}";
                        rpcMethodName = $"{localRpcInterfaceCode}.{rpcMethodName}";
                    }
```

/cc @stephentoub 😄

Reply:
Or why not just:

```cs
                    if (rpcInterfaceCode.HasValue)
                    {
                        methodName = $"{rpcInterfaceCode}.{method.Name}";
                        rpcMethodName = $"{rpcInterfaceCode}.{rpcMethodName}";
                    }
```

Reply:
For the latter, just because nullable value types are a bit less efficient in interpolated strings. Probably not a big deal, though.

For the former, seems like a stylistic preference.

Reply:
It's exactly the same code, but more CSharpy. 😄